content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -193,7 +193,7 @@ sudo apt-mark hold kubelet kubeadm kubectl
 cat <<EOF > /etc/yum.repos.d/kubernetes.repo
 [kubernetes]
 name=Kubernetes
-baseurl=https://packages.cloud.google.com/yum/repos/kubernetes-el7-\$basearch
+baseurl=https://packages.cloud.google.com/yum/repos/kubernetes-el7-x86_64
```

Comment:
hi, isn't `$basearch` defined on your side?
what errors are you getting?

cc @sftim 

/sig cluster-lifecycle
/priority awaiting-more-evidence

Reply:
I don't see there's anything to fix. PR #20291 still seems like the right fix here. However, I didn't test that fix thoroughly.